opentelemetry-semantic-conventions/scripts/templates/registry/rust/attribute.rs.j2

```diff
@@ -0,0 +1,24 @@
+{%- import 'macros.j2' as attr_macros -%}
+// DO NOT EDIT, this is an auto-generated file
+//
+// If you want to update the file:
+// - Edit the template at scripts/templates/registry/rust/attributes.rs.j2
+// - Run the script at scripts/generate-consts-from-spec.sh
```

Comment:
From CI prespective I think we need a CI check to make sure the commited version of `attribute.rs` is consistent with the one generate from script? I am thinking about the case where I accidentially typed something in the generated files and commit it.

Reply:
I agree. One option would be to update Weaver to generate a checksum (e.g., SHA256) for the generated code and store it in a file alongside the generated code. This would give us everything needed to create a CI check to ensure that no manual modifications have been made to the code in the repository, assuming we’ve also prevented rust fmt from altering the generated code. I can add this option to Weaver in a future release if the Rust Client SDK community is interested.

Reply:
Just created this feature request in Weaver. https://github.com/open-telemetry/weaver/issues/362

Reply:
Thanks. This can be a follow up and not a blocker for the PR. @TommyCpp Do you agree to resolve this ?

Reply:
Yeah, a follow up sounds good